webapp/channels/src/components/with_tooltip/create_tooltip.tsx

```diff
@@ -3,55 +3,93 @@
 
 import React from 'react';
 import type {ComponentProps} from 'react';
-import type {MessageDescriptor} from 'react-intl';
+import {type MessageDescriptor} from 'react-intl';
 
 import RenderEmoji from 'components/emoji/render_emoji';
-import {ShortcutKey, ShortcutKeyVariant} from 'components/shortcut_key';
 import Tooltip from 'components/tooltip';
 
-import {getStringOrDescriptorComponent} from './utils';
+import {TooltipShortcutSequence, type ShortcutDefinition} from './shortcut';
+import {getAsFormattedMessage} from './utils';
+
+type EmojiStyle = 'inline' | 'large' | undefined;
 
 export type CommonTooltipProps = {
     id: string;
-    title: string | MessageDescriptor;
-    hint?: string | MessageDescriptor;
-    shortcut?: string[];
+    title: string | MessageDescriptor | React.ReactElement;
+    hint?: string | MessageDescriptor | React.ReactElement;
```

Comment:
Oh yeah, that's true. Looking at the HTML, there are actually `b` tags for a few words inside the tooltip, but since we bold that whole title line of the tooltip, those `b` tags don't do anything visually. I'll check with Abhijit to see if that's considered as a bug or if it's something we can keep the behaviour as-is since those `b` tags are the only reason that we need `ReactElement` here

Reply:
Actually, I'm just going to do this since the current appearance matches the design file anyway